Strategies/Account.py

```diff
@@ -29,9 +31,14 @@ def sell(self, share_price, shares_to_sell=None):
             shares_to_sell = self.shares
         
         self.balance += shares_to_sell * share_price
-        self.shares -= shares_to_sell
+        self.shares = 0 #-= shares_to_sell
```

Comment:
Is this okay? I know I changed it but I just wanted to make sure. Technically I think we lose a small amount of money?

Reply:
we really should be rounding to the nearest pip